packages/editor/src/components/post-title/index.native.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { TextInput } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { decodeEntities } from '@wordpress/html-entities';
+import { ENTER } from '@wordpress/keycodes';
+import { withInstanceId, compose } from '@wordpress/compose';
+
+class PostTitle extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.onChange = this.onChange.bind( this );
+		this.onSelect = this.onSelect.bind( this );
+		this.onUnselect = this.onUnselect.bind( this );
+		this.onKeyDown = this.onKeyDown.bind( this );
+		this.redirectHistory = this.redirectHistory.bind( this );
+
+		this.state = {
+			isSelected: false,
+		};
+	}
+
+	handleFocusOutside() {
+		this.onUnselect();
+	}
+
+	onSelect() {
+		this.setState( { isSelected: true } );
+		this.props.clearSelectedBlock();
```

Comment:
I am guessing you are planning to do a separate PR but just wanted to note, clearSelectedBlock() also will need to be propagated to the store.

Reply:
thanks! 🎉